src/API/Site/Controllers/MerchantCenter/AccountReviewController.php

```diff
@@ -62,10 +55,10 @@ public function register_routes(): void {
 	protected function get_review_read_callback(): callable {
 		return function ( Request $request ) {
 			// TODO: Temporary. Implement this after Google finishes new API implementation.
+			// Statuses: APPROVED, UNDER_REVIEW, DISAPPROVED, WARNING, BLOCKED
 			$response = [
-				'status'      => $this->request_review->is_allowed() ? RequestReview::REVIEW_STATUS_DISAPPROVED : RequestReview::REVIEW_STATUS_BLOCKED,
-				'nextAttempt' => $this->request_review->get_next_attempt(),
-				'issues'      => [
+				'status' => 'BLOCKED',
```

Comment:
❓

If we removed `nextAttempt` would be convenient to update the schema properties See line: https://github.com/woocommerce/google-listings-and-ads/blob/df4f68d0ed4cec1e5ca765be9cc3623e263ba233/src/API/Site/Controllers/MerchantCenter/AccountReviewController.php#L88

Reply:
Good you noticed this @jorgemd24 

Fixed here: https://github.com/woocommerce/google-listings-and-ads/pull/1256/commits/f371b5286e060f0436cf8e5cfaae1d555521bf19